File: Modul-3/Soal-5/PRAK305-2410817210008-AhmadUlyani.c
#include <stdio.h>
int main (){
    int totaldetik, jam, menit, detik;
    scanf("%d", &totaldetik);
    jam = totaldetik / 3600;
    menit = (totaldetik % 3600) / 60;
    detik = totaldetik % 60;
    if (jam >= 24){
        int hari = jam / 24;
        jam = jam % 24;
        printf("%d hari %02d:%02d:%02d",hari, jam, menit, detik);
    }
    else {
        printf("%02d:%02d:%02d", jam, menit, detik);
    } 

    return 0;
}

File: Modul-3/Soal-5/PRAK305-2410817210008-AhmadUlyani.py
totaldetik= int(input())
jam = totaldetik // 3600
menit = (totaldetik % 3600) // 60
detik = totaldetik % 60

if jam >= 24:
    hari = jam // 24
    jam = jam % 24
    print(f"{hari} hari {jam:02}:{menit:02}:{detik:02}")

else:
    print(f"{jam:02}:{menit:02}:{detik:02}")